content/node-runners/mysterium-cli.md

```diff
@@ -2,3 +2,137 @@
+
+### How to use the CLI
+
+To start open your terminal and execute:
+```bash
+myst cli
+```
+
+Once started your terminal window should change to the `myst cli` interface.
+To get help and see a full list of possible commands execute:
+```bash
+help 
+```
+
+Each command has an output, if the output does not start with `[ERROR]` or `[WARNING]` consider that command a success.
+
+Commands can be autocompleted using the `tab` key on your keyboard.
+
+### Initial setup 
+
+If you have already installed a Mysterium node but have no access to the NodeUI to complete the onboarding process, you'll have to unlock and register your identity with Mysterium CLI.
+Go ahead and fire up the cli application and follow the steps below. 
+
+#### Check your identity 
+> Command: identities list
+
+In order to identify yourself in the network you must have an identity which was created during the installation process. Use the command below to print your node identity address to your terminal.
+
+```bash
+identities new
+```
+
+#### Unlock the new identity 
+> Command: identities unlock [identity] 
+
+In order to use the created identity it has to be unlocked first.
+
+**Unlocking has to be done every time we launch the `myst cli` and want to use the created identity.**
+
+For the sake of simplicity let's say we've received an identity: `0x4570fe47a49af9ae9bd76f029818413ea18620000`,
+which we'll use in all later steps.
+
+To unlock it we'd have to execute:
+```bash
+identities unlock 0x4570fe47a49af9ae9bd76f029818413ea18620000
+```
+
+#### Register your identity
+> Command: identities register [identity] [stake]
+
+After creating and unlocking the identity it must be registered so that Mysterium services are aware
+about it and it's balance, to do that execute the `identities register` command.
+
+```bash
+identities register 0x4570fe47a49af9ae9bd76f029818413ea18620000 0
+```
+
+#### Claim your node into my.mysterium.network
+```bash
+mmn <your-api-key>
+```
+**Note**: You can find and copy your MMN API key in your [MMN profile](https://my.mysterium.network/user/profile).
+
+
+#### Set payout wallet address
+
+```bash
+identities set-payout-address <providerIdentity> <beneficiary>
+```
+
+There are two types of wallets that you can use: 
+
+
+>  Exchange wallets (HitBTC and Bittrex deposit addresses are supported only); 
+
+>  Direct Blockchain wallets (Metamask, Trust wallet, Atomic, MEW, MyCrypto, Trezor, Ledger ...). 
+
+ 
+MYST is a standard ERC20 token, if wallet supports ERC-20 tokens (has possibility to add any custom ERC-20 token), then MYST can be added there.
+
+##### MYST token details: 
+
+> Contract: 0x4Cf89ca06ad997bC732Dc876ed2A7F26a9E7f361
+
+> Symbol: MYST
+
+> Decimals: 18
+
+#### Get payout wallet address
+
+Ensure that beneficiary wallet address has been successfully changed:
+
+```bash
+identities get-payout-address <providerIdentity>
+```
+
+### Managing node service
+
+##### To start WireGuard service we'd have to execute:
```

Comment:
This is started by default. We should either clarify that or not talk about it at all

Reply:
Clarified.